entity-web/src/main/java/eu/europeana/entity/web/controller/ResolveController.java

```diff
@@ -119,7 +119,10 @@ public ResponseEntity<String> getXmlHeaderEntity(
     private ResponseEntity<String> createResponse(String type, String identifier, FormatTypes outFormat,
             String contentType, HttpServletRequest request) throws HttpException {
         try {
-            verifyReadAccess(request);
+            if (isAuthEnabled()) {
```

Comment:
we should mode this check to the parent implementation (api-commons BaseRestController)